### PR TITLE
Changed counting of incoming and processing rate from created to processed date

### PR DESCRIPTION
**Pull request by jnmrclmbsjse**
This PR includes;
1. Updated storekeeper task table. Added `date_last_processed`.
2. Changed processing calculation from created date to processed date.
3. Updated unit test

### CODE REVIEW

File: src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php [hunk 1]
```diff
@@ -200,7 +207,7 @@ public static function countTasksByCreatedDateTimeRange($start, $end): int
     {
         global $wpdb;
 
-        $select = static::prepareCreatedDateTimeRangeSelect($start, $end);
+        $select = static::prepareProcessedDateTimeRangeSelect($start, $end);
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php:210]: the `public static function countTasksByCreatedDateTimeRange($start, $end): in` is not correct anymore

File: src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php [hunk 2]
```diff
@@ -188,7 +195,7 @@ public static function getExecutionDurationSumByCreatedDateTimeRange($start, $en
...
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php:198]: the name `public static function getExecutionDurationSumByCreatedDateTimeRange($start, $end): ?float` is not correct anymore

File: src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php (hunk 1 above)
Comment (ikulis) [reply src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php:210]: this function was also used in counting of the incomming tasks

File: tests/unit/Tools/TaskRateCalculatorTest.php [hunk 3]
```diff
@@ -23,8 +23,8 @@ public function testEmptyTasks()
     public function testTaskIncomingRate()
     {
         // make a task
-        $this->createTaskWithCreatedDate(1, '1970-01-01 01:30:00');
-        $this->createTaskWithCreatedDate(2, '1970-01-01 01:45:00');
+        $this->createTaskWithProcessedDate(1, '1970-01-01 01:30:00');
```
Comment (ikulis) [inline tests/unit/Tools/TaskRateCalculatorTest.php:26]: this test should not change incomming counting was correct

File: src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php (hunk 1 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php:233]: Reverted to created date.

File: src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php (hunk 2 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php:207]: Changed name of method to `getExecutionDurationSumByProcessedDateTimeRange`.

File: tests/unit/Tools/TaskRateCalculatorTest.php (hunk 3 above)
Comment (jnmrclmbsjse) [reply tests/unit/Tools/TaskRateCalculatorTest.php]: Reverted to created date.

File: tests/unit/Tools/TaskRateCalculatorTest.php [hunk 4]
```diff
@@ -35,8 +35,8 @@ public function testTaskIncomingRate()
 
     public function testTaskProcessedRate()
     {
-        $task1 = $this->createTaskWithCreatedDate(1, '1970-01-01 01:30:00');
-        $task2 = $this->createTaskWithCreatedDate(2, '1970-01-01 01:45:00');
+        $task1 = $this->createTaskWithProcessedDate(1, '1970-01-01 01:30:00');
```
Comment (ikulis) [inline tests/unit/Tools/TaskRateCalculatorTest.php:38]: Ad few created, processed tasks before 1hour

**Pull request merged**